MessengerOnFileForLinux/CMakeLists.txt

```diff
@@ -31,8 +31,8 @@ set(MODULE_MessageFramework         ${CMAKE_SOURCE_DIR}/MessageFramework/src)
 ################################################################################
 #####                       CREATE ENVIRONMENT CMAKE PART                  #####
 ################################################################################
-set(clear_enviroment "/scripts/clear_enviroment.sh")
-set(create_enviroment "/scripts/create_enviroment.sh")
+set(clear_enviroment "/scripts/local_scripts/clear_enviroment.sh")
```

Comment:
a może by zrobić skrypt, który wywołuje zawsze clear i create, a dla przypadków gdy chcesz tlyko wyczyścić to wtedy dodajesz parametr i leci tylko clear?

Reply:
twoja sugestia pokazała własnie że ten skrypt nie nazywa się najlepiej, bo on nie czyści środowiska tylko je usuwa :)

Reply:
dobra, ale nadal moja uwaga jest aktualna. Zawsze przed testowaniem chcesz usunąć stare śmieci, ale po testowaniu czasami też chcesz je usunąć. Ale nigdy nie chcesz zrobić create_env bez pierw czyszczenia.

Reply:
tylko ze samo clear ci nic nie da bo ci usunie cale srodowisko.
Zrobie jednen skrypt ktory bedize robil create i remove i zostawie clear tylko ze będzie on teraz czyścił a nie usuwał. ok ?

Reply:
clear mi da to, że po testowaniu messengera i odejściu od tego nie chce na swoim kompie mieć śmieci w /var/tmp :)

Reply:
ok